### ISSUE TITLE
Extract "Pangalink" payment gateway to a gem

### ISSUE THREAD

**Issue by artur-beljajev** [internetee/registry#646]
Might be connected to #642.

**Comment by teadur** [comment]
I think it makes more sense to use already existing gem:
https://github.com/Voog/ipizza

**Comment by artur-beljajev** [comment]
Agree! I will review them.

**Comment by maciej-szlosarczyk** [comment]
I don't think that using `ipizza` gems is a good idea. It feels abandonded. It hasn't been updated for over 3 years and there are few issues and PRs that haven't been addressed since. It will stop working sooner rather than later, and then we're back to square one.

Additionally, `ipizza_rails` lacks any kind of tests, which makes both Ruby and Rails upgrades more difficult than they should be.
